components/google-cloud/storage/download_dir/component.yaml

```diff
@@ -1,6 +1,7 @@
 name: Download from GCS
 inputs:
-- {name: GCS path, type: URI}
+- {name: GCS path, type: GCSPath}
+- {name: Recurse, type: Boolean, default: True}
```

Comment:
Are you sure "Recurse" is needed for the download_dir component? Doesn't dir imply the recursive download?

Reply:
I think you're right: disabling recursive copy would download only the directory and its "immediate" contents, but not the sub-directories. I don't have a use case for this, and the additional knob is probably confusing.. I'll remove it.